libsrc/telestrat/keyboard.s

```diff
@@ -0,0 +1,14 @@
+; jede jede@oric.org 2017-01-22
+
+    .export         _key
+				
+    .include        "zeropage.inc"		
+    .include        "telestrat.inc"
+
+; char key(void);
+		
+.proc _key
+    BRK_TELEMON XRDW0 ; read keyboard
```

Comment:
i removed this file too. I will have a look. I did this to share headers/functions between cc65 and Oric C compiler (osdk) that is why i did this. But, i understand that cc65 must keep standard C librairies.